tests/testthat/test-correspondenceanalysis.R

```diff
@@ -166,10 +188,29 @@ test_that("Bubble charts",
                 expect_error(CorrespondenceAnalysis(x.with.labels, output = "Bubble Chart", show.gridlines = FALSE))
                 bsizes = x.with.labels[,1]
                 expect_error(print(CorrespondenceAnalysis(x.with.labels, output = "Bubble Chart", bubble.size = bsizes)), NA)
+                expect_error(CorrespondenceAnalysis(x.with.labels, output = "Bubble Chart", bubble.size = bsizes, row.names.to.remove = "Coke"), NA)
+                expect_error(CorrespondenceAnalysis(t(x.with.labels), output = "Bubble Chart", transpose = TRUE,
+                    bubble.size = bsizes, row.names.to.remove = "Coke"), NA)
+
+                expect_error(CorrespondenceAnalysis(t(x.with.labels), output = "Bubble Chart", transpose = TRUE,
+                    bubble.size = bsizes, column.names.to.remove = "Coke"), NA)
+                expect_error(CorrespondenceAnalysis(t(x.with.labels), output = "Bubble Chart", transpose = TRUE,
+                    bubble.size = bsizes, row.names.to.remove = "Coke"), NA)
+                expect_error(CorrespondenceAnalysis(x.with.labels, output = "Bubble Chart", bubble.size = bsizes, transpose = TRUE),
+                    "Bubble sizes: Missing values for 'Kids', 'Teens'")
+
+                expect_warning(CorrespondenceAnalysis(x.with.labels, output = "Bubble Chart",
+                    bubble.size = c(Extra = 1, More = 2, bsizes)), "Bubble sizes for 'Extra', 'More' were ignored.")
+                expect_error(CorrespondenceAnalysis(x.with.labels, output = "Bubble Chart", bubble.size = bsizes[-1]),
+                    "Bubble sizes: Missing values for 'Coke'")
+                expect_error(CorrespondenceAnalysis(x.with.labels, output = "Bubble Chart",
+                    bubble.size = c(bsizes[-1], 'coke ' = 2)), NA)
+
                 names(bsizes)[2] = "Dog"
-                expect_error(print(CorrespondenceAnalysis(x.with.labels, output = "Bubble Chart", bubble.size = bsizes)), "The bubble sizes must contain the same names")
+                expect_error(print(CorrespondenceAnalysis(x.with.labels, output = "Bubble Chart", bubble.size = bsizes)), "Bubble sizes: Missing values for 'V'")
```

Comment:
I think the error message could be improved. Tim said in the jira: "To use a Bubble Chart, the table of Bubble sizes needs to include all the row categories used in the analysis. The following are missing: Coke, Pepsi"

Reply:
Remember to deal with singular/plural cases.